IronPythonTests/Setups.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
```

Comment:
Setups.cs (instead of Setup.cs)
Was that to simply the diff viewer ?

Reply:
nah, this is a type ;) will fix.